recipes/example/meta.yaml

```diff
@@ -1,34 +1,44 @@
-{% set version = "1.0" %}
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "3.8.2" %}
 
 package:
-  name: example
-  version: {{ version }}
+    name: simplejson
+    version: {{ version }}
 
 source:
-  git_url: https://github.com/conda-forge/staged-recipes.git
-  git_tag: 24507f6c6b2d2e03caee550e13830df15d791751
+    # It is preferable to get the package "closer" to the source. For example use GitHub instead PyPI whenever possible.
+    fn: simplejson-{{ version }}.tar.gz
+    url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+    # md5 is OK but sha256 is desirable.
+    sha256: 0041407316fbd35786883436c9ffaba4b799569174f22ee54e758f72249b1c67
 
 build:
-  number: 0
+    number: 0
+    # Don't a bld.bat nor build.sh when the installation is 1-2 liners. Use the `build/script` instead.
```

Comment:
Actually, since this is what the example uses, better to phrase it the other way round:

"If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key"